base/io.jl

```diff
@@ -167,24 +167,50 @@ The text is assumed to be encoded in UTF-8.
 readuntil(filename::AbstractString, args...) = open(io->readuntil(io, args...), filename)
 
 """
-    readline(stream::IO=STDIN)
-    readline(filename::AbstractString)
+    readline(stream::IO=STDIN, chomp::Bool=false)
+    readline(filename::AbstractString, chomp::Bool=false)
```

Comment:
Yes, but really that should be much easier than what you've done until now. If you really can't, I'll do it for you.